packages/accounts-controller/tsconfig.build.json

```diff
@@ -3,7 +3,8 @@
   "compilerOptions": {
     "baseUrl": "./",
     "outDir": "./dist",
-    "rootDir": "./src"
+    "rootDir": "./src",
+    "skipLibCheck": true
```

Comment:
Ignoring type errors from @metamask/rpc-methods

node_modules/@metamask/rpc-methods/dist/types/restricted/getLocale.d.ts:46:113 - error TS2344: Type 'void' does not satisfy the constraint 'RestrictedMethodParameters | null'.

export declare function getImplementation({ getLocale }: GetLocaleMethodHooks): (_args: RestrictedMethodOptions<void>) => Promise<string>;
Will revert once it is fixed